packages/webdriverio/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "baseUrl": ".",
     "outDir": "./build",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "types": ["webdriverio"]
```

Comment:
@Fabianopb FYI: you have to update `tsconfig.prod.json` as well.

Fixed it in https://github.com/webdriverio/webdriverio/commit/903a25575cb0e9f9a553265249fcb9c8f50c18ae

Reply:
Good point, thanks for the fix!